docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -167,30 +169,29 @@ Install the SkyPilot Helm chart with the following command:
     $ WEB_USERNAME=skypilot
     $ WEB_PASSWORD=yourpassword
     $ AUTH_STRING=$(htpasswd -nb $WEB_USERNAME $WEB_PASSWORD)
+    $ helm repo update
     $ helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
     $   --namespace $NAMESPACE \
     $   --create-namespace \
     $   --set ingress.authCredentials=$AUTH_STRING
 
+If you configured any cloud credentials in the previous step, make sure to enable them by adding the relevant flags (e.g., ``--set awsCredentials.enabled=true``) to the command.
+
 The ``--namespace`` flag specifies which namespace to deploy the API server in, and ``--create-namespace`` will create the namespace if it doesn't exist.
 
 To install a specific version, pass the ``--version`` flag to the ``helm upgrade`` command (e.g., ``--version 0.1.0``).
 
-If you configured any cloud credentials in the previous step, make sure to enable them by adding the relevant flags (e.g., ``--set awsCredentials.enabled=true``) to the command.
-
 .. tip::
 
     You can configure the password for the API server with the ``WEB_PASSWORD`` variable.
 
 .. tip::
 
-    If you already have a Kubernetes secret containing basic auth credentials, you can use it directly by setting ``ingress.authSecret`` instead of ``ingress.authCredentials``:
+    If you already have a Kubernetes secret containing basic auth credentials for the nginx ingress, you can use it directly by setting ``ingress.authSecret`` instead of ``ingress.authCredentials``:
 
     .. code-block:: console
 
-        $ helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
-        $   --namespace $NAMESPACE \
-        $   --create-namespace \
+        $ # helm upgrade --install ... \
         $   --set ingress.authSecret=my-existing-auth-secret
 
     The secret must be in the same namespace as the API server and must contain a key named ``auth`` with the basic auth credentials in htpasswd format.
```

Comment:
Can we add a cross reference to Step 3 in the section `Updating the API server`?